scipy-1.0/paper.tex

```diff
@@ -352,45 +352,6 @@ \subsection*{SciPy Begins}
 the Space Telescope Science Institute (STScI), which was in charge of
 Hubble Space Telescope science operations, decided to replace their
 custom scripting language and analysis pipeline with Python\cite{STScI-slither}.
-
-As SciPy, the algorithms library of the ecosystem, began attracting the attention
```

Comment:
This section is great, but it's not really part of the history of SciPy 1.0. If we have to trim, this seems a natural place to start.